### DISCUSSION TITLE
How to ignore specific headers? (I read the doc!)

### DISCUSSION THREAD

**Discussion by theLSB** [cppinclude/cppinclude#13] [Q&A]
Hi,

great idea for the tool!
However, I seem to not fully understand how to filter out results.
In my case, I have Qt in my project, and I want the tool to ignore Qt includes.
The following does not seem to help:
```
$cppinclude --report=most_impact --ignore_files=Q/.*
Start initialization project ...
Start analyze sources ...
Start report results ...
Most impact files:
1 : "QObject" impact on 1497 file(s)
Included by:
```
It seems to me (the documentation formulation is not so clear) that ignore_files and ignore_dir options are excluding them from scanning for includes, but not for ignoring them if they are included.
Is that the case?

How can I have cppinclude scan my project, but ignore the Qt headers (or any others for that matter) included by my project?

Thanks!

**Comment by cppinclude** [comment]
Hello,

Sorry for late answer. Example for ignore Qt libraries : --ignore_files=^Q.* . This library use std::regexp
